app/models/miq_schedule_worker/jobs.rb

```diff
@@ -16,6 +16,10 @@ def miq_server_worker_log_status
     queue_work(:class_name  => "MiqWorker", :method_name => "log_status_all", :task_id => "log_status", :server_guid => MiqServer.my_guid, :priority => MiqQueue::HIGH_PRIORITY)
   end
 
+  def miq_audit_vm_total
```

Comment:
why the `miq` prefix on this method name?  Is that a standard in this file?  /cc @Fryguy

Reply:
I would suggest a simpler name, such as `audit_managed_resources`

Reply:
I was using the same convention (although missed the _server) as the rest of the file "miq_server_worker_log_status" and "miq_server_status_update" which are the two methods above. My guess is that it is a way to discern where the job is coming from. So the method should be "miq_server_audit_managed_resources"

Reply:
If you look through the file, you will see many methods without `miq_server`.  `miq_server` refers to the MiqServer object (which is associated with the appliance).  I advise we use the name `audit_managed_resources` here.